LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2014-2022 Chart.js Contributors
```

Comment:
[nit] should be 2017
```suggestion
Copyright (c) 2017-2022 Chart.js Contributors
```

![image](https://user-images.githubusercontent.com/39033624/186170518-73bc5215-b60f-4182-9fd9-26c5f54b2dd6.png)